ctriface/manual_cleanup_test.go

```diff
@@ -288,21 +294,9 @@ func TestParallelPhasedSnapLoad(t *testing.T) {
 			go func(i int) {
 				defer vmGroup.Done()
 				vmID := fmt.Sprintf("%d", i+vmIDBase)
```

Comment:
Why you removed shutting down the instances? it seems you changed the tested scenario. if you think you need another scenario, add a separate test